pom.xml

```
@@ -4,13 +4,77 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
-    <groupId>com.epam.tc.hw0</groupId>
+    <groupId>com.epam.tc.hw1</groupId>
```

Let's leave it like this
```suggestion
    <groupId>com.epam.tc</groupId>
```